### PR TITLE
Change dart:convert "UTF8" -> "utf8".

### PR DESCRIPTION
**Pull request by munificent**
The former will be removed in Dart 2, so this gets it ready for that.
The new names were already introduced in a previous dev version.

### CODE REVIEW

**Review by lyceel — APPROVED** [review]

**Comment by munificent** [comment]
Thanks! I don't have commit rights, so would you mind merging this in?

Also, updating the pubspec and publishing 1.0.2 will make it easier for me to roll it inside Google. Thank you! :)

**Comment by lyceel** [comment]
Sorry, I missed the "Confirm Merge" button after clicking "Merge"

**Comment by munificent** [comment]
Thanks!

**Pull request merged**
